src/styles/blocks/page.scss

```
@@ -0,0 +1,13 @@
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  &__calendar {
+    margin: 0 auto;
+    padding-top: 10px;
+    width: #{$cell-size * 7 + 10px};
+    
+  }
```

```suggestion
    width: #{$cell-size * 7 + 10px};    
  }
```
Лишняя пустая строка. Есть еще парочка таких по файлам. 
Не совсем поняла, что за 10рх в ширине календаря. Возможно, если это значение может меняться, его стоит тоже задать переменной.